Makefile

```diff
@@ -31,7 +31,7 @@ test-watch: $(node_modules) build-buf
 lint: $(node_modules) build-buf
 	npm run lint
 	npm run typecheck
-        // TODO(RSDK-5407): We can stop ignoring `@viamrobotics/rpc` once build issues are resolved in the latest version.
+	# TODO(RSDK-5407): We can stop ignoring `@viamrobotics/rpc` once build issues are resolved in the latest version.
```

Comment:
Be careful about putting comments inside the body of Makefile recipes; they will be sent to the shell. It'll usually be ok, because a # signifies a comment in bash, too, but every now and again you can really weird issues from it (or at least confusing CI logs). Generally, I try to keep comments outside of recipes and put them above the target, instead

Reply:
Good call. As this is temporary (will get removed when we merge in the PR to update RPC), I'll leave it for now. But good practice